etc/containerpilot.json

```diff
@@ -85,7 +85,7 @@
                   "--cron",
                   "--domain", "{{ .ACME_DOMAIN }}",
                   "--hook", "/etc/dehydrated/hook.sh",
-                  "--config", "/etc/dehydrated/config"],
+                  "--config", "/etc/dehydrated/config.{{ .ACME_ENV }}"],
```

Comment:
Is there a good way to default this to `staging` in the case it's omitted from `docker-compose.yml`? cc @tgross